src/Symfony/Component/DomCrawler/Field/FormField.php

```diff
@@ -99,7 +99,7 @@ public function getValue()
     /**
      * Sets the value of the field.
      *
-     * @param string $value The value of the field
+     * @param string|array|bool|null $value The value of the field
```

Comment:
Should `getValue()` be updated accordingly? 
Also the implementation does not seem to support arrays

Reply:
nvm